setup.py

```diff
@@ -74,6 +74,7 @@ def get_readme():
         'pandas',
         'plotly',
         'scipy>=1.6',
+        'pints @ git+https://github.com/pints-team/pints.git#egg=pints-latest',
```

Comment:
Do you remember if this was necessary, or if we couldnt just use pints without the git @ .